frontend/components/sections/BannersSection/SingleBanner.tsx

```diff
@@ -40,38 +30,43 @@ export function SingleBanner({ id, banner }: SingleBannerProps) {
                />
             </Box>
          )}
-         <Wrapper>
-            <Box textAlign="center">
-               {banner.label && (
-                  <Text color="white" mb="3" fontSize="sm">
-                     {banner.label}
-                  </Text>
-               )}
-               {banner.title && (
-                  <SectionHeading color="white" mb="2.5">
-                     {banner.title}
-                  </SectionHeading>
-               )}
-               {banner.description && (
-                  <SectionDescription
-                     richText={banner.description}
-                     color="white"
-                     maxW="750px"
-                     mx="auto"
-                  />
-               )}
-               {banner.callToAction && (
+         <Wrapper position="relative" textAlign="center">
+            {banner.label && (
+               <Text color="white" mb="3" fontSize="sm">
+                  {banner.label}
+               </Text>
+            )}
+            {banner.title && (
+               <SectionHeading color="white" mb="2.5">
+                  {banner.title}
+               </SectionHeading>
+            )}
+            {banner.description && (
+               <SectionDescription
+                  richText={banner.description}
+                  color="white"
+                  maxW="750px"
+                  mx="auto"
+               />
+            )}
+            {banner.callToAction && (
+               <Box
+                  position="absolute"
+                  w="full"
+                  bottom="-20"
+                  left="50%"
+                  transform="translateX(-50%)"
+               >
```

Comment:
If we want all content including the button to vertically center, we should remove this Box component. It has `position: absolute` applied, so it's removed from the page stacking order.

Reply:
The call-to-action button was excluded from the "centered" content before, and it continues to be excluded in this pull. I did my best to keep the spacing the same.

### Main

<img src="https://github.com/iFixit/react-commerce/assets/52104630/4ab1c6d3-8474-4e9d-ab74-96c2e60908d6" height="200px">

### Feature

<img src="https://github.com/iFixit/react-commerce/assets/52104630/1495e6fc-76ad-4d07-980d-8bc6de3b7e3c" height="200px">